src/types/field/emailField.ts

```diff
@@ -1,20 +1,15 @@
-import { IField, IFieldSchema } from './baseField'
+import {
+  AutoReplyOptions,
+  BasicField,
+  EmailFieldBase,
+} from '../../../shared/types/field'
 
-export type AutoReplyOptions = {
-  hasAutoReply: boolean
-  autoReplySubject: string
-  autoReplySender: string
-  autoReplyMessage: string
-  includeFormSummary: boolean
-}
+import { IFieldSchema } from './baseField'
 
-export interface IEmailField extends IField {
-  autoReplyOptions: AutoReplyOptions
-  isVerifiable: boolean
-  hasAllowedEmailDomains: boolean
-  allowedEmailDomains: string[]
-}
+export { AutoReplyOptions }
 
+export type IEmailField = EmailFieldBase
 export interface IEmailFieldSchema extends IEmailField, IFieldSchema {
+  fieldType: BasicField.Email
   isVerifiable: boolean
```

Comment:
doesn't the base type already contain `isVerifiable`?

Reply:
Same reasoning, IFieldSchema has `isVerifiable?: boolean` and we want to assert that it is no longer optional (following IEmailField)